tooling/template-base/components/CatalogAndAggreg/Aggregations.tsx

```diff
@@ -0,0 +1,123 @@
+import React, { ReactNode, useState } from 'react';
+import { useCatalog } from '@thoughtindustries/catalog';
+import clsx from 'clsx';
```

Comment:
Can you add this to `template-base`'s `package.json`? It works now because the workspace dependencies are linked, but would have issues when installing the `template-base` and running it on its own. Can install by running `npm i clsx -w tooling/template-base` from the `helium` root.

Reply:
hey Rob, just pushed there